new_samples/README.md

```diff
@@ -0,0 +1,32 @@
+# Cadence Samples
+These are some samples to demonstrate various capabilities of Cadence client and server.  You can learn more about cadence at:
+* Cadence: https://github.com/uber/cadence
+* Cadence Client: https://github.com/uber-go/cadence-client
+
+## Prerequisite
+0. Install Cadence CLI. See instruction [here](https://cadenceworkflow.io/docs/cli/).
+1. Run the Cadence server: https://github.com/uber/cadence/blob/master/README.md
+2. Register the `cadence-samples` domain: https://cadenceworkflow.io/docs/cli/#quick-start or https://github.com/uber/cadence/blob/master/tools/cli/README.md
```

Comment:
```suggestion
2. Register the `cadence-samples-domain` domain: https://cadenceworkflow.io/docs/cli/#quick-start or https://github.com/uber/cadence/blob/master/tools/cli/README.md
```
since the const elsewhere has the `-domain` suffix.

or, to match the original readme, maybe `samples-domain` is better?